### ISSUE TITLE
fix(rule): DailyLossLimitRule·TotalExposureLimitRule 매도(손절) 허용 및 알림 전환

### ISSUE THREAD

**Issue by joshua-jingu-lee** [joshua-jingu-lee/ante#715]
## 배경

`DailyLossLimitRule`과 `TotalExposureLimitRule`이 손실/노출 한도 초과 시 매도(손절)까지 차단하여, 포지션 정리가 불가능하고 손실이 오히려 확대되는 역설적 상황이 발생한다.

업계 조사 결과, "매도 포함 전면 차단"을 기본 동작으로 채택하는 시스템은 없다:
- **Freqtrade**: 진입만 차단, 기존 포지션은 자체 SL/TP로 관리
- **QuantConnect**: 강제 청산 (포지션 정리는 허용)
- **IB / FINRA**: 신규 매수 차단, 매도는 허용

**방향**: 진입(매수)만 차단, 매도는 허용, 사용자에게 알림하여 판단을 맡긴다.

ref: `docs/temp/spec-gap-004-loss-limit-blocks-sell.md`

## 스펙 수정

`docs/specs/rule-engine.md` 168-169행 — 이미 반영 완료:
- `DailyLossLimitRule`: "계좌 중지" → "추가 매수 차단 (매도는 허용), 사용자 알림"
- `TotalExposureLimitRule`: "거래 제한" → "추가 매수 차단 (매도는 허용), 사용자 알림"

### 1. `DailyLossLimitRule.evaluate()` — 매도 즉시 PASS + 액션 변경

**파일**: `src/ante/rule/global_rules.py`

```python
class DailyLossLimitRule(Rule):
    """일일 손실 한도 초과 시 추가 매수 차단."""

    def evaluate(self, context: RuleContext) -> RuleEvaluation:
        max_daily_loss = self.config.get("max_daily_loss_percent", 0.05)

        # 매도(손절)는 항상 허용 — 포지션 정리 경로를 보장
        if context.side == "sell":
            return RuleEvaluation(
                rule_id=self.rule_id,
                rule_name=self.name,
                result=RuleResult.PASS,
                action=RuleAction.LOG,
                message="Sell order allowed during loss limit",
            )

        if context.daily_pnl < 0 and context.prev_day_total_asset > 0:
            daily_loss_percent = abs(context.daily_pnl) / context.prev_day_total_asset

            if daily_loss_percent > max_daily_loss:
                return RuleEvaluation(
                    rule_id=self.rule_id,
                    rule_name=self.name,
                    result=RuleResult.REJECT,      # BLOCK → REJECT
                    action=RuleAction.NOTIFY,       # HALT_ACCOUNT → NOTIFY
                    message=(
                        f"Daily loss limit exceeded: "
                        f"{daily_loss_percent:.2%} > {max_daily_loss:.2%}. "
                        f"Buy orders blocked. Sell orders are still allowed."
                    ),
                    metadata={
                        "daily_loss_percent": daily_loss_percent,
                        "max_daily_loss_percent": max_daily_loss,
                        "daily_pnl": context.daily_pnl,
                        "prev_day_total_asset": context.prev_day_total_asset,
                    },
                )

        return RuleEvaluation(
            rule_id=self.rule_id,
            rule_name=self.name,
            result=RuleResult.PASS,
            action=RuleAction.LOG,
            message="Daily loss within limits",
        )
```

**변경 요점**:
- `context.side == "sell"` → 즉시 PASS (`UnrealizedLossLimitRule`과 동일 패턴)
- `RuleResult.BLOCK` → `REJECT` (주문 거부만, 계좌 중지 안 함)
- `RuleAction.HALT_ACCOUNT` → `NOTIFY` (사용자에게 알림, 판단은 사용자 몫)

### 2. `TotalExposureLimitRule.evaluate()` — 동일 패턴 적용

**파일**: `src/ante/rule/global_rules.py`

```python
def evaluate(self, context: RuleContext) -> RuleEvaluation:
    # 매도(포지션 축소)는 항상 허용
    if context.side == "sell":
        return RuleEvaluation(
            rule_id=self.rule_id,
            rule_name=self.name,
            result=RuleResult.PASS,
            action=RuleAction.LOG,
            message="Sell order allowed during exposure limit",
        )
    ...  # 이하 기존 로직 (매수만 검사)
```

### 3. 테스트

**파일**: `tests/unit/test_rule.py`

추가 케이스:
- `DailyLossLimitRule`: 손실 한도 초과 + `side="sell"` → PASS
- `DailyLossLimitRule`: 손실 한도 초과 + `side="buy"` → REJECT + NOTIFY (BLOCK/HALT_ACCOUNT 아님)
- `TotalExposureLimitRule`: 노출 한도 초과 + `side="sell"` → PASS

## 수정 대상 파일

- `src/ante/rule/global_rules.py` — DailyLossLimitRule, TotalExposureLimitRule 매도 허용 + 액션 변경
- `tests/unit/test_rule.py` — 매도 허용 테스트 추가

**Comment by joshua-jingu-lee** [comment]
## 보완: 알림 메시지 정의 추가

`docs/specs/rule-engine.md`에 "알림 이벤트 정의 (Notification Events)" 섹션을 추가하여, 룰 위반 시 발행하는 `NotificationEvent`의 구체적인 메시지 포맷을 정의했습니다.

### 추가 구현 사항

#### `DailyLossLimitRule` — NotificationEvent 메시지

```
⚠️ 일일 손실 한도 초과
계좌 {account_id}의 일일 손실률이 한도를 초과했습니다.
손실률: {daily_loss_percent:.2%} (한도: {max_daily_loss:.2%})
당일 손익: {daily_pnl:,.0f}원
추가 매수가 차단됩니다. 매도(손절)는 허용됩니다.
보유 포지션을 확인하고 필요한 조치를 취해주세요.
```

#### `TotalExposureLimitRule` — NotificationEvent 메시지

```
⚠️ 총 노출 한도 초과
계좌 {account_id}의 총 포지션 노출이 한도를 초과했습니다.
노출: {expected_exposure:,.0f}원 / 한도: {exposure_limit:,.0f}원 (총 자산의 {max_exposure_percent:.0%})
추가 매수가 차단됩니다. 매도(포지션 축소)는 허용됩니다.
포지션 배분을 확인하고 필요한 조치를 취해주세요.
```

#### 구현 위치

`src/ante/rule/engine.py`의 `_execute_actions()` 메서드에서 `RuleAction.NOTIFY` 처리 시, `RuleEvaluation.metadata`의 값들을 사용하여 위 포맷의 메시지를 생성한다. `NotificationEvent`의 속성:
- `level`: `"warning"`
- `category`: `"rule"`
- `title`: `"일일 손실 한도 초과"` 또는 `"총 노출 한도 초과"`
- `message`: 위 포맷의 전체 메시지

스펙 참조: `docs/specs/rule-engine.md` "알림 이벤트 정의" 섹션, `docs/specs/bot.md` "알림 이벤트 정의" 섹션 (포맷 일관성)

**Comment by joshua-jingu-lee** [comment]
🤖 **구현 착수**
- 담당 에이전트: @backend-dev
- 변경 대상: src/ante/rule/global_rules.py, tests/unit/test_rule.py
- base 브랜치: main

**Comment by joshua-jingu-lee** [comment]
🤖 **코드 리뷰: APPROVE**
- PR: #716
- 체크리스트 A~F 전체 통과
